text/0013-schema-enums.md

```diff
@@ -0,0 +1,154 @@
+# Enums in Schemas
+
+## Related issues and PRs
+
+- Reference Issues: 
+- Implementation PR(s): 
+
+## Timeline
+
+- Start Date: 2023-06-30
+- Date Entered FCP:
+- Date Accepted:
+- Date Landed:
+
+## Summary
+
+Add support for enumerations of `bool`, `string`, and `long` types in schemas, and use this information when doing policy validation and schema-based-parsing.
+
+## Basic example
+
+Maintaining backward compatibility with JSON schema as much as possible, we can match its enum feature. For example:
+
+```json
+"type": "Record",
+  "attributes": {
+    "color": {
+      "type": "String",
+      "enum" : ["red", "blue", "green"]
```

Comment:
I'd prefer `choices` to `enum` as the field name.

Reply:
That sounds sensible to me.  Or `options`

Reply:
We want to follow what JSON schema does. That's `enum`. https://json-schema.org/understanding-json-schema/reference/enum

Reply:
Oh, in that case yes let's do that